pymarl/config/algs/qvmix_uc_100.yaml

```diff
@@ -0,0 +1,29 @@
+# --- QVmix specific parameters ---
+
+# use epsilon greedy action selector
+action_selector: "epsilon_greedy"
+epsilon_start: 1.0
+epsilon_finish: 0.05
+epsilon_anneal_time: 100000
+
+runner: "episode"
+#batch_size_run: 6
+
+batch_size: 32
```

Comment:
Did you test with batch_size 64 as the others?

Reply:
Not yet. I will do it.

Reply:
I just changed it now.